tests/ssl-opt.sh

```diff
@@ -14235,79 +14235,79 @@ run_test    "Client Handshake defragmentation (5)" \
 requires_openssl_tls1_3
 run_test    "Server Handshake defragmentation (512)" \
             "$P_SRV debug_level=4 " \
-            "$O_NEXT_CLI -max_send_frag 512 " \
+            "$O_NEXT_CLI -max_send_frag 512 -cert $DATA_FILES_PATH/server5.crt -key $DATA_FILES_PATH/server5.key" \
```

Comment:
Unless I'm mistaken, you also want to add `auth_mode=optional` (or required) to the server command line, as by default the server doesn't ask for a client certificate, and the client doesn't send one unless the server asked.

Note: this certificate is 548 bytes, so that's good.